packages/fxa-settings/src/pages/CookiesDisabled/en.ftl

```diff
@@ -0,0 +1,4 @@
+cookies-disabled-header = Cookies and local storage are required
```

Comment:
One more comment:
Other recent PRs recreating pages in react have been adding group level comments ([example](https://github.com/mozilla/fxa/blob/f3b3db60206919fecff9331d615d6ca73a20c3dc/packages/fxa-settings/src/components/ConfirmWithLink/en.ftl#L1)) to provide context for both localizers and for those reviewing the post-concatted fluent files. The IDs in this are fairly straight forward so not necessarily required, but pointing it out nonetheless.